testplan/web_ui/testing/src/Nav/TagList.js

```diff
@@ -53,6 +53,7 @@ TagList.propTypes = {
 const styles = StyleSheet.create({
   tags: {
     'margin-right': '.4em',
+    'background-color': '#6a95c3'
```

Comment:
move the color definition to https://github.com/morganstanley/testplan/blob/main/testplan/web_ui/testing/src/Common/Styles.js or 
https://github.com/morganstanley/testplan/blob/main/testplan/web_ui/testing/src/Common/defaults.js